### PR TITLE
Block photos link

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

File: frontend/src/components/BlocCf/index.jsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-      <div className="photoDiv 	fifthPhoto">
-        <h3 className="titlePhoto">FORMATION</h3>
-      </div>
+      <Link to="amateurs">
+        <div className="photoDiv fourthPhoto">
+          <h3 className="titlePhoto">TERRITORIALES</h3>
+        </div>
+      </Link>
+      <Link to="formation">
+        <div className="photoDiv 	fifthPhoto">
```
Comment (LuigiMenez) [inline frontend/src/components/BlocCf/index.jsx:13]: Why a big space?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Why a big space?

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**